launch-script/src/main.rs

```diff
@@ -293,6 +293,52 @@ fn main() {
                         .help("Proposal for vote")
                 )
             )
+            .subcommand(SubCommand::with_name("create-start-evm")
+                .about("Create proposal for Neon EVM start")
+                .arg(
+                    Arg::with_name("collateral-pool-base")
+                        .long("collateral-pool-base")
+                        .required(true)
+                        .takes_value(true)
+                        .value_name("COLLATERAL_POOL_BASE")
+                        .help("Collateral pool base")
+                )
+                .arg(
+                    Arg::with_name("evm-loader")
```

Comment:
Take this value from wallet `neon_evm_program_id` (https://github.com/neonlabsorg/neon-spl-governance/blob/13cdf8e09f81a57d9a60893091f3982f0126063e/launch-script/src/wallet.rs#L30).

Try to answer the question: what happens if the user specifies different EVM_LOADER_ADDRESS on different calls `launch-script`? To eliminate problems I collect all meaningful values in one configuration file.

Reply:
Fixed